src/System.Drawing.Common/src/System/Drawing/SolidBrush.cs

```diff
@@ -2,44 +2,26 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
 namespace System.Drawing
 {
-    using System.Diagnostics;
-    using System.Drawing.Internal;
-    using System.Runtime.InteropServices;
-
-    /// <include file='doc\SolidBrush.uex' path='docs/doc[@for="SolidBrush"]/*' />
-    /// <devdoc>
-    ///    <para>
-    ///       Defines a brush made up of a single color. Brushes are
-    ///       used to fill graphics shapes such as rectangles, ellipses, pies, polygons, and paths.
-    ///    </para>
-    /// </devdoc>
-    public sealed class SolidBrush : Brush, ISystemColorTracker
+    public sealed class SolidBrush : Brush
+#if FEATURE_SYSTEM_EVENTS
```

Comment:
Why are you adding this define?  If we don't need the code, we should remove it entirely.  If we do need it, we shouldn't have it conditionally compiled.